app/views/users/show.html.erb

```diff
@@ -1,5 +1,16 @@
 <% content_for :title, current_user == @user ? "Your account" : "#{@user.name.presence || @user.email}’s account" %>
 
+<% if current_user.support? %>
+  <% content_for :breadcrumbs, govuk_breadcrumbs(breadcrumbs: {
+    "Users (#{@user.organisation.name})" => users_organisation_path(@user.organisation),
```

Comment:
`DLUHC Users` sounds a bit better than `Users (DLUHC)` to. me, but that's preference so no need to action

Reply:
Will run this by Olivia :)